src/samaware/templates/samaware/talk_overview.html

```diff
@@ -161,11 +162,7 @@ <h3>{% translate 'Submitter Notes' %}</h3>
                 <em>{% translate 'None' %}</em>
                 {% endif %}
 
-                <h3>{% translate 'RT Ticket' %}</h3>
```

Comment:
Since I don't have a setup to test this locally –  are the signal source sexpected to add a `<h3>` heading?

Reply:
yes.